backend/coreapp/compilers.py

```diff
@@ -736,6 +736,11 @@ def preset_from_name(name: str) -> Optional[Preset]:
         MWCC_247_92,
         "-lang=c++ -g -Cpp_exceptions off -RTTI off -fp hard -fp_contract on -O4,p -maxerrors 1 -str reuse,pool,readonly -char unsigned -enum int -use_lmw_stmw on -inline off",
     ),
+    Preset(
+        "Mario Kart Double Dash!!",
```

Comment:
It looks like the official non-stylized name is `Mario Kart: Double Dash` with the colon and without the exclamation points. I think we should use that one instead. Otherwise, looks good!

Reply:
I believe it's Mario Kart: Double Dash!! but honestly i don't mind the name :P

Reply:
the stylized name includes the semicolons, but the non-stylized name doesn't https://www.wikidata.org/wiki/Q5721431
I just think the exclamation points look kinda loud/out of place on the site